golem-api-grpc/proto/golem/worker/v1/worker_service.proto

```diff
@@ -114,6 +116,21 @@ message InvokeAndAwaitResponse {
   }
 }
 
+message InvokeAndAwaitJsonRequest {
```

Comment:
is this still used?

Reply:
yes, added it so these things (like the json-invoke drop error) can be tested through "regular" integration tests not just via cli.